src/utils.ts

```diff
@@ -39,6 +39,10 @@ export const PAGE_TEMPLATE_FOLDER = 'portal:template-folder';
 
 export const SITE_CONTENTTYPE_NAME = 'portal:site';
 
+export const LOCALE_HEADER = 'locale';
+export const LOCALES_HEADER = 'locales';
+export const DEFAULT_LOCALE_HEADER = 'default-locale';
```

Comment:
Why do we need to export these?

Reply:
I though about anyone consuming the request to know the header names